map_test.go

```diff
@@ -1044,6 +1056,49 @@ func TestIterateEmptyMap(t *testing.T) {
 	}
 }
 
+func TestDrainEmptyMap(t *testing.T) {
```

Comment:
Is this test necessary? You could just try another Drain at the end of the other tests below.

Reply:
I wanted to keep the same separation as for `TestIterateEmptyMap` https://github.com/cilium/ebpf/blob/c6205528d1f7c29552021af9bcd2f049dd92cabe/map_test.go#L1003

Let me know.